docker.js

```diff
@@ -23,14 +23,19 @@ function login(username, password, server) {
 /**
  * Builds the image described by the Dockerfile and tags it locally.
  * @param {string} tag - The local tag to use for the built image.
- * @param {string} dockefilePath - The path to the Dockerfile.
- *   (Can be a directory or the path to a file to use as Dockerfile.)
+ * @param {string} dockefile - The Dockerfile path.
```

Comment:
I think this can still be a directory to use.
I would also be good to explain whether the context changes the path that the dockerfile is look for in or not.